java-squid/src/main/java/org/sonar/java/resolve/FirstPass.java

```diff
@@ -255,6 +256,13 @@ public void visitClass(ClassTree tree) {
     scan(tree.superClass());
     scan(tree.superInterfaces());
     scan(tree.members());
+    //Register default constructor
+    if(tree.is(Tree.Kind.CLASS) && classEnv.scope.lookup("<init>").isEmpty()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "<init>" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)